libraries/AP_GroundEffectControl/AP_GroundEffectController.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+   
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+   
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//	Code by Sebastian Quilter
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_GroundEffectController.h"
+#include <AP_AHRS/AP_AHRS.h>
+
+extern const AP_HAL::HAL& hal;
+
+constexpr uint32_t RESET_TIMEOUT_MICROS{1000000};
+
+const AP_Param::GroupInfo GroundEffectController::var_info[] = {
```

Comment:
We should have a enable param, both to hide the group and allow to be actively enabled. Probably also need a `#define` to make the feature optional.

Reply:
done